src/Core/Joins.h

```diff
@@ -0,0 +1,95 @@
+#pragma once
+
+namespace DB
+{
+
+/// Join method.
```

Comment:
Also, we have this enum used to improve readability in some cases.

https://github.com/ClickHouse/ClickHouse/blob/24aaac59ca207fd3dcd1bea34a67acc86fba58d0/src/Interpreters/TableJoin.h#L45-L49

Reply:
Updated.